modules/help_editor/jsx/help_editor.js

```diff
@@ -0,0 +1,135 @@
+import FilterForm from 'FilterForm';
+import formatColumn from './columnFormatter';
+
+/**
+ * Help Editor Archive Page.
+ *
+ * Serves as an entry-point to the module, rendering the whole react
+ * component page on load.
+ *
+ * Renders Help Editor main page consisting of FilterTable and
+ * DataTable components.
+ *
+ * @author Alex Ilea
+ * @version 1.0.0
+ *
+ * */
+class HelpEditor extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      filter: {}
```

Comment:
A state of `Data: {}` should be placed here, purely for documentation purposes. On that note, this may be a good time to change `Data` to `data` since it breaks lowerCamelCase convention.